src/probability/stopping.lean

```diff
@@ -1291,4 +1292,197 @@ by { ext x, rw stopped_value, by_cases hx : x ∈ s; simp [hx] }
 
 end piecewise_const
 
+section hitting
+
```

Comment:
That section could become big very quickly. I think you should create a new file instead.